components/nexus-ehcache/src/main/java/org/sonatype/nexus/cache/EhCacheManagerProvider.java

```diff
@@ -43,24 +49,48 @@
     extends ComponentSupport
     implements Provider<CacheManager>
 {
-  private final URI uri;
+  private static final String CONFIG_FILE = "ehcache.xml";
+
+  private static final String DEFAULT_RESOURCE = "ehcache-default.xml";
+
+  private volatile CacheManager cacheManager;
 
-  private CacheManager cacheManager;
+  // TODO: Consider if we want to lazily create the instance on first get?  Unsure if it matters or not though.
```

Comment:
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png) Complete the task associated to this TODO comment. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1135)